src/main/java/com/sprint/mission/discodeit/repository/file/FileChannelRepository.java

```diff
@@ -2,59 +2,71 @@
 
 import com.sprint.mission.discodeit.entity.Channel;
 import com.sprint.mission.discodeit.repository.ChannelRepository;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
 
 import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.*;
 
+@Repository
 public class FileChannelRepository implements ChannelRepository {
-    private final String FILE_PATH="channel.ser";
+    private final Path DIRECTORY;
+    private final Path FILE_PATH;
```

Comment:
```suggestion
    private final Path directory;
    private final Path filePath;
```

상수가 아니라면 카멜케이스를 써보는건 어때요?